src/private/MDMViewControllerTransitionContext.m

```diff
@@ -52,7 +53,8 @@ - (nonnull instancetype)initWithTransition:(nonnull id<MDMTransition>)transition
 - (NSTimeInterval)transitionDuration:(id<UIViewControllerContextTransitioning>)transitionContext {
   // TODO(featherless): Expose a TransitionWithTiming protocol that allows the transition to
   // customize this value.
-  return 0.35;
+  return [_transition transitionDuration];
```

Comment:
This is where you'd add a runtime check for `respondsToSelector:@selector(transitionDurationWithContext:)`. If it's not implemented then we'd use the default value of 0.35.